change timeseries to time_series in BehavioralEvents

kevinalexbrown opened the pull request:
fixes #500

kevinalexbrown committed:
change timeseries to time_seris in BehavioralEvents

kevinalexbrown committed:
fixed behavioral events test in test_behavior.py

bendichter committed:
Merge branch 'dev' into behavioral_bugfix

oruebel commented:
@ajtritt can you have a quick look at this, I think PupilTracking and BehavioralTimeSeries may have the same issue.

ajtritt reviewed (changes requested):
@oruebel is correct. PupilTracking also has the problem. @kevinalexbrown can you do the same fix to PupilTracking?

kevinalexbrown commented:
Yes

On Tue, May 1, 2018 at 2:27 PM Andrew Tritt <notifications@github.com>
wrote:

> *@ajtritt* requested changes on this pull request.
>
> @oruebel <https://github.com/oruebel> is correct. PupilTracking also has
> the problem. @kevinalexbrown <https://github.com/kevinalexbrown> can you
> do the same fix to PupilTracking?
>
> —
> You are receiving this because you were mentioned.
>
>
> Reply to this email directly, view it on GitHub
> <https://github.com/NeurodataWithoutBorders/pynwb/pull/501#pullrequestreview-116655172>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAzjEkjolfWJuspc11o5H8bC0WATpIXLks5tuKkYgaJpZM4TrH_R>
> .
>
-- 
Kevin

oruebel commented:
@ajtritt does BehavioralTimeSeries have the same issue or only PupilTracking?

kevinalexbrown commented:
Just PupilTracking, I fixed BehavioralTimeSeries when I fixed
BehavioralEvents

On Tue, May 1, 2018 at 3:22 PM Oliver Ruebel <notifications@github.com>
wrote:

> @ajtritt <https://github.com/ajtritt> does BehavioralTimeSeries have the
> same issue or only PupilTracking?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/NeurodataWithoutBorders/pynwb/pull/501#issuecomment-385763613>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAzjEjaZuRXmi0nfqFbWzyixowDfNj7lks5tuLX_gaJpZM4TrH_R>
> .
>
-- 
Kevin

kevinalexbrown committed:
Fixed PupilTracking in behavior.py

kevinalexbrown committed:
Merge branch 'behavioral_bugfix' of github.com:kevinalexbrown/pynwb into behavioral_bugfix

ajtritt committed:
Merge branch 'dev' into behavioral_bugfix
